packages/compiler/src/core/helpers/type-name-utils.ts

```diff
@@ -182,7 +182,16 @@ function getInterfaceName(iface: Interface, options: TypeNameOptions | undefined
 }
 
 function getOperationName(op: Operation, options: TypeNameOptions | undefined) {
-  return `${getNamespacePrefix(op.namespace, options)}${getIdentifierName(op.name, options)}`;
+  let opName = getIdentifierName(op.name, options);
+  if (op.node.templateParameters.length > 0) {
+    // template
+    const params = op.node.templateParameters.map((t) => getIdentifierName(t.id.sv, options));
+    opName += `<${params.join(", ")}>`;
+  }
```

Comment:
@timotheeguerin , about the prefix of operation, I think it should be namespaceName.interfaceName.operationName. Please let me know if it's not the case. Some test cases are impacted by this change. please help to confirm. thanks.

And I also found we dont have template arguments shown when hovering on it either. But I notice we have logic handling the instantiated template (in getInterfaceName function), so i am not sure whether it's by design for not showing interface's template parameter, please help to confirm or it's a bug. thanks.